scripts/arnie-quotes.coffee

```diff
@@ -30,10 +30,10 @@ arnie_quotes = [
 module.exports = (robot) ->
     robot.hear /(^|\s)arnie(\s|$|[\W])/ig, (msg) ->
         val = msg.random odds
-        if val > 50
+        if val > 0 # Set Arnie to reply 100% for now
             msg.send msg.random arnie_quotes
 
     robot.hear /(^|\s)arnold(\s|$|[\W])/ig, (msg) ->
         val = msg.random odds
-        if val > 50
+        if val > 0 # Set Arnie to reply 100% for now
```

Comment:
I left the percentages in case we ever go back, which I think is likely. It's easy enough to remove though. I'll make those changes and test right quick.